.gitignore

```diff
@@ -67,3 +67,4 @@ tags
 /tests/*.sh
 /tests/phpt.*
 /C:\\Windows\\Temp\\remote-log4.txt
+/.idea/
```

Comment:
You can git-ignore `/.idea/` in all your repositories by adding it to a global .gitignore file ([example](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore))

Reply:
Indeed, I'll do so. Since I got pretty messy, I think I'll flatten the commits in a cleaner PR following the bugfix rules more properly (forking xdebug_2_7 branch, using issue1665 name, making a single commit, and removing this gitignore line)